scss/variables/colors.js

```diff
@@ -24,7 +25,8 @@ chassis.colors = {
 			"base": "#3F51B5",
 			"light": "#5C6BC0",
 			"dark": "#3949AB",
-			"darker": "#303F9F"
+			"darker": "#303F9F",
+			"notification": "#3F51B5"
```

Comment:
The "default" color's base and default notification's color differ hence I decided to add a notification key in each of the color instead of just adding in one and creating a special case of default in mixin.scss